docs/migrations/v2-to-v3.md

```diff
@@ -47,7 +47,8 @@ app.UpgradeKeeper.SetUpgradeHandler("v3",
         
         ...
 
-        return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+        vm[icatypes.ModuleName] = 1
```

Comment:
@colin-axner added this during the QA session we had to test the upgrade. I added it here now because I guess we need it, right?

Reply:
I don't think so? I think this skips ICA [InitGenesis](https://docs.cosmos.network/v0.44/core/upgrade.html)

We do probably need to add the [store loading documentation](https://docs.cosmos.network/v0.44/core/upgrade.html#add-storeupgrades-for-new-modules)